### DISCUSSION TITLE
Getting window position on startup

### DISCUSSION THREAD

**Discussion by MixmasterFresh** [hecrj/iced#1296] [Q&A]
I'm trying to implement a custom window drag bar(I'm using `decorations: false` in window settings), and I think I don't have the information necessary to do it without something a little hacky. Here's where I'm at:

1. the `Move` action for windows takes an absolute position of where to place it on the screen
2. It seems like if I use the `position: position::Centered` then there is no way for me to get the starting coordinates of the window for which I could apply offsets
3. I could force the window position to a particular starting location(using `position::Specific`) and use that to create offsets from, but I'd probably have to use (0,0) to be safe for all devices, which is technically fine(but I don't like it)

Am I right here? Is there any way I can get direct access to the winit window object? Or would it be better for me to submit a PR to add something like a `MoveBy` that takes a logical offset instead of an absolute position? Or is there a way I can do this without a library code change? 

Thanks a ton in advance

**Comment by hecrj** [comment]
This particular use case (custom window drag bar) is one that we haven't considered yet.

If we want to tackle it, I believe we will need to get any new features through [the RFC process](https://github.com/iced-rs/rfcs) to precisely state what use cases are we tackling (i.e. the scope) and iterate on the best design for a solution.
